test/WebJobs.Script.Tests/Artifacts/ApiHub/ApiHubTestHelper.cs

```diff
@@ -0,0 +1,81 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
```

Comment:
What does the "Artifacts" mean? You might just have a Tests/ApiHub folder and move all this ApiHub stuff in there?